app/data/doc-collections/index.ts

```diff
@@ -45,6 +46,7 @@ export const testCollections = {
 export const docCollections = {
   ...(includeTestingDocCollections ? testCollections : {}),
   cadence,
+  "cadence/style-guide": cadence__styleguide,
```

Comment:
Specify the base URL for the repo's `docs` folder content. 
In this case the files in your repo will appear under `/cadence/style-guide/your-doc`